### PR TITLE
use message to open duckplayer from overlay

### PR DESCRIPTION
**Pull request by shakyShane**
https://app.asana.com/0/1201141132935289/1207702853516473/f

This just removes a single instance where the page location was updated by the JS. 

**Note** This needs testing on macOS + Windows

- [ ] macOS
- [ ] windows

### CODE REVIEW

**Comment by shakyShane** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/992?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#993** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/993?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#992** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/992?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#991** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/991?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @shakyShane and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by Bunn — DISMISSED** [review]
on macOS tested:
- with overlay, tap on watch in duck player
- with overlay, tap on watch on youtube
- without overlay, opening directly 
working as expected, with the difference that the overlay correctly calls the handler now

File: integration-test/playwright/page-objects/duckplayer-overlays.js [hunk 1]
```diff
@@ -346,16 +346,26 @@ export class DuckplayerOverlays {
...
-                if (f.url().startsWith('duck')) resolve(f.url())
-            })
-        })
+        const action = () => this.page.getByRole('link', { name: 'Watch in Duck Player' }).click()
 
-        await this.page.getByRole('link', { name: 'Watch in Duck Player' }).click()
+        await this.build.switch({
+            'apple-isolated': async () => {
+                await action()
+                await this.duckPlayerLoadsFor('123')
```
Comment (shakyShane) [inline integration-test/playwright/page-objects/duckplayer-overlays.js:354]: on macos now, we're just sending a message

**Review by mgurgel — APPROVED** [review]
👍

**Pull request merged**